src/mlpack/core/data/string_cleaning.hpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file string_cleaning.hpp
+ * @author jeffin sam
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_STRING_CLEANING_HPP
+#define MLPACK_CORE_DATA_STRING_CLEANING_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <boost/range/algorithm/remove_if.hpp>
+#include <boost/algorithm/string/classification.hpp>
+#include <boost/utility/string_view.hpp>
+#include <mlpack/core/data/tokenizer/strtok.hpp>
+#include <boost/functional/hash.hpp>
+
+std::string stopwords = 
+#include "mlpack/core/data/stopwords.txt"
+;
+
+class Hasher {
+ public:
+  std::size_t operator()(boost::string_view str) const {
+    return boost::hash_range<const char*>(str.begin(), str.end());
+  }
+};
+ 
```

Comment:
Now , Since standalone function, can't migrate the class above . Any other workaround.